jetstream/cli.py

```diff
@@ -44,6 +44,7 @@ def setup_logger(
 
 
 RECOGNIZED_EXPERIMENT_TYPES = ("pref", "addon", "message", "v6")
+OUTCOMES_DIR = "outcomes"
```

Comment:
The assumption here is that all outcome snippets are stored in `outcomes/` in jetstream-config